src/euphorie/client/tests/test_consultancy.py

```diff
@@ -110,6 +113,21 @@ def test_request_validation(self):
                     self.traversed_session.session.consultancy.account,
                     self.consultant,
                 )
+                events = (
+                    self.session.query(SessionEvent)
+                    .filter(
+                        SessionEvent.session_id == self.traversed_session.session.id
+                    )
+                    .order_by(SessionEvent.time.desc())
+                    .all()
+                )
+                self.assertEqual(len(events), 1)
```

Comment:
If you take the query above and call `.one()` instead of `.all()`, you are automatically asserting that you have one matching row.

Reply:
That would change the test failure to a database error. I don't think that's what I want.

Reply:
That's the point, you will not even assert... anyway, that's fine